app/helpers/application_helper.rb

```diff
@@ -59,6 +59,18 @@ def room_panorama( room )
     end
   end
 
+  def api_status
+    if ApiUpdateLog.find_by(created_at: Time.zone.now.beginning_of_day..Time.zone.now.end_of_day).nil?
+      return "failed"
+    elsif ApiUpdateLog.find_by(created_at: Time.zone.now.beginning_of_day..Time.zone.now.end_of_day).status == "error"
+      return 'error'
+    end
+  end
+
+  def api_log_text
```

Comment:
do you think a √ for nilClass should go here to prevent throwing an error?

Reply:
I added the check. There is no way to test this because api_log_text is displayed if api_status exists and == 'error'.
The ApiUpdateLog model's status and result fields are not nil. So there is no way to create a record with nill api_log_text (result field).